_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,235 @@
+`--smtppassword` | The SMTP password.| For example, `SMTP_PASSWORD`. | OPENSEARCH_SMTP_PASSWORD
+`--subject` | Email Subject text encased in quotes. | Can be any string. The default is "This is an email containing your dashboard report". | OPENSEARCH_EMAIL_SUBJECT
+`-h`, `--help` | Display list of optional arguments from the command line. | N/A
+
+
+
+## Use environment variables with Reporting CLI
+
+Instead of explicitly providing values in the command line, you can save them as environment variables. The Reporting CLI reads environment variables from the current directory inside the project.
+
+To set the environment variables on Linux, use the following command. 
+
+```
+export NAME=VALUE
+```
+
+Each line should have the format `NAME=VALUE`.
+Each line that starts with a hashtag (#) is considered as a comment.
+Quotation marks (") don't get any special handling.
+
+Values from the command line argument have higher priority than the environment file. For example, if you add filename as *test* in *.env* file and also add `--filename report` command option, the generated report's name will be *report*.
+{: .note }
+
+### Example: Request a PNG report with environment variables set
+
+The following command requests a dashboard report with basic authentication in png format.
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --format png --auth basic --credentials admin:admin
+```
+
+Upon success, the report will download to the current directory.
+
+### Use Amazon SES to request an email with a report attachment
+
+To use Amazon SES as the email transport mechanism, the following prerequisites apply:
+
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily. The sender's email address must be verified on [Amazon SES](https://aws.amazon.com/ses/).
+- If a URL contains an exclamation point (!), then the history expansion needs to be disable temporarily.
+ [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) is required to interact with Amazon SES.
+ [Configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) basic settings used by AWS CLI.
+- SES transport requires the `ses:SendRawEmail` role.
+
+```
+{
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "ses:SendRawEmail",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+The following command requests an email with the report attached:
+
+```
+opensearch-reporting-cli --url https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d --transport ses --from <sender_email_id> --to <recipient_email_id>
+```
+
+The following command uses default values for all other options. You can also set `OPENSEARCH_FROM`, `OPENSEARCH_TO`, and `OPENSEARCH_TRANSPORT` in .env file and use following command.
```

Comment:
```suggestion
The following command uses default values for all other options. You can also set `OPENSEARCH_FROM`, `OPENSEARCH_TO`, and `OPENSEARCH_TRANSPORT` in the .env file and use the following command:
```

Reply:
Confirm that the first article I added makes sense (that it should not be "an" or "your").